Main.java

```diff
@@ -1,15 +1,55 @@
+import java.util.Scanner;
+
 /**
- *
- * @author 
+ * This program will let the user input a pizza with their desired size and will
+ * display a message bassed off of what they ordered.
+ * 
+ * @author Gormi3815
  */
 public class Main {
 
   /**
    * The method that is executed when you hit the run button.
+   * 
    * @param args the command line arguments
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+    // I'm inputting a few of the integer along with their coralating questions so I
+    // can get all of the information that I will need
+    System.out.println("What size of pizza would you like 1-20 small, 21-40 medium 40+ large?");
+
+    double diameter = input.nextInt();
+    double diameter_cost = diameter * .5;
+    System.out.println("How many pizzas would you like?");
+    double ammountOfPizza = input.nextInt();
+    double costPerPizza = ammountOfPizza * .99;
+    double labour = ammountOfPizza * .75;
+    double subtotal = labour + diameter_cost + costPerPizza;
+    double tax = subtotal * .13;
```

Comment:
The numbers here should really be in constants. Right now, they are just "magic numbers" that are floating around.